Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/bazel-public/bazel:7.4.1 AS builder
```

Comment:
We also need to update the [.bazelversion](https://github.com/freifunkMUC/wgkex/blob/main/.bazelversion) file

Reply:
Done.